src/main.rs

```diff
@@ -292,13 +288,14 @@ async fn main() -> Result<()> {
     // Create the event loop and TCP listener we'll accept connections on.
     let local_addr = matches
         .value_of("address")
-        .ok_or(anyhow!("missing --address arg"))?;
+        .ok_or_else(|| anyhow!("missing --address arg"))?;
     let rfb_addr: std::net::SocketAddr = matches
         .value_of("rfb-server")
-        .ok_or(anyhow!("missing --rfb-server arg"))?
+        .ok_or_else(|| anyhow!("missing --rfb-server arg"))?
         .parse()?;
     let enable_audio = matches.is_present("enable-audio")
-        || std::env::var("VNC_ENABLE_EXPERIMENTAL_AUDIO").unwrap_or(String::from("")) != "";
+        || std::env::var("VNC_ENABLE_EXPERIMENTAL_AUDIO").unwrap_or_else(|_| String::from(""))
```

Comment:
haha I think the `unwrap_or` would be fine here since Rust doesn't allocate for an empty string. c'mon clippy!

could also do a:
```suggestion
        || std::env::var("VNC_ENABLE_EXPERIMENTAL_AUDIO").map(|s| !s.is_empty()).unwrap_or(false)
```

Reply:
oh you know what? this could use `String::new()` instead of `String:from("")`. why this wasn't suggested is a mystery to me.

Reply:
Oh yeah, that seems like it would be a pretty simple clippy lint too 🤔